charts/connect/templates/connect-deployment.yaml

```diff
@@ -78,6 +94,22 @@ spec:
             {{- toYaml .Values.connect.sync.resources | nindent 12 }}
           ports:
             - containerPort: 8081
+          {{- if .Values.connect.probes.readiness }}
+          readinessProbe:
+            httpGet:
+              path: /health
+              port: 8080
+            initialDelaySeconds: 15
+          {{ end }}
+          {{- if .Values.connect.probes.liveness }}
+          livenessProbe:
+            httpGet:
+              path: /health
```

Comment:
The api and sync containers have a `/heartbeat` endpoint that should be used for the `livenessProbe`

Reply:
Sure thing. Changed and pushed.